langchain/src/runnables/remote.ts

```diff
@@ -325,4 +335,73 @@ export class RemoteRunnable<
     });
     return IterableReadableStream.fromReadableStream(stream);
   }
+
+  async *streamLog(
+    input: RunInput,
+    options?: Partial<CallOptions>,
+    streamOptions?: Omit<LogStreamCallbackHandlerInput, "autoClose">
+  ): AsyncGenerator<RunLogPatch> {
+    const [config, kwargs] =
+      this._separateRunnableConfigFromCallOptions(options);
+    const stream = new LogStreamCallbackHandler({
+      ...streamOptions,
+      autoClose: false,
+    });
+    const { callbacks } = config;
+    if (callbacks === undefined) {
+      config.callbacks = [stream];
+    } else if (Array.isArray(callbacks)) {
+      config.callbacks = callbacks.concat([stream]);
+    } else {
+      const copiedCallbacks = callbacks.copy();
+      copiedCallbacks.inheritableHandlers.push(stream);
+      config.callbacks = copiedCallbacks;
+    }
+    const getExtraOptionsFromInput = () => {
+      const optionKeys = [
+        "diff",
+        "include_names",
+        "include_types",
+        "include_tags",
+        "exclude_names",
+        "exclude_types",
+        "exclude_tags",
+      ];
+      const extraOptions: Record<string, unknown> = {};
+      for (const key of optionKeys) {
+        const keyAsKeyof = key as keyof RunInput;
+        if (input[keyAsKeyof] !== undefined && input[keyAsKeyof] !== null) {
+          extraOptions[key] = input[keyAsKeyof];
+        }
+      }
```

Comment:
in python we force `diff=True` I think for stream log

Reply:
Added that as default, and it'll throw if you try and pass `false`.